playbooks/10 - SP - GDPR Framework/Check for SLA violation.json

```diff
@@ -127,30 +133,30 @@
     "routes": [
         {
             "@type": "WorkflowRoute",
-            "name": "Configuration -> Fetch Open Compliance Record",
-            "targetStep": "\/api\/3\/workflow_steps\/5dc8e4ca-f1e6-4083-900d-f386baa71e7b",
-            "sourceStep": "\/api\/3\/workflow_steps\/1ff4ec35-2d6a-4c5e-bfa2-0c5abca19ed4",
+            "name": "Start -> Configuration",
+            "targetStep": "\/api\/3\/workflow_steps\/1ff4ec35-2d6a-4c5e-bfa2-0c5abca19ed4",
+            "sourceStep": "\/api\/3\/workflow_steps\/27d5b17b-21de-4899-b1d4-b23ac209b993",
             "label": null,
             "isExecuted": false,
-            "uuid": "01776d04-ad2c-44f1-bdec-1f6d0cb39c41"
+            "uuid": "4c1c6c15-6f53-4962-bf33-05744043165b"
         },
         {
             "@type": "WorkflowRoute",
-            "name": "Find Open Compliance Record -> Update Breached Records",
-            "targetStep": "\/api\/3\/workflow_steps\/eae18941-aeb3-49d9-8311-ace38a6a3324",
-            "sourceStep": "\/api\/3\/workflow_steps\/5dc8e4ca-f1e6-4083-900d-f386baa71e7b",
+            "name": "Find Open Comliance Records -> Send Reminder for SLA Breach",
```

Comment:
Comliance --> Compliance

Reply:
@cs-suyog-jinturkar, this is actually the `WorkflowRoute` name. I previously misspelt a step name and corrected it later. But even though we rename step, the `WorkflowRoute` name remains the same

Reply:
Remove the route and re-add it